sdcm/nemesis.py

```diff
@@ -7020,7 +6715,7 @@ class FreeTierSetMonkey(SisyphusMonkey):
     def __init__(self, *args, **kwargs):
         # skip SisyphusMonkey __init__ to not repeat build disruption logic, but still we want to run Nemesis class __init__
         super(SisyphusMonkey, self).__init__(*args, **kwargs)  # pylint: disable=bad-super-call
-        self.build_list_of_disruptions_to_execute(nemesis_selector=['free_tier_set'])
+        self.build_list_of_disruptions_to_execute(nemesis_selector='free_tier_set')
```

Comment:
why not utilizing `NEMESIS_SELECTOR` class attr (applies also to SlaNemeses, ZeroTokenSetMonkey)?

Reply:
It could be used here as well, will fix